x/incentive/spec/03_keeper.md

```diff
@@ -0,0 +1,49 @@
+<!--
+order: 3
+-->
+
+# Keepers
+
+## Reward pool address for LPs
+
+There's constant reward pool address per liquidity pool id which will be distributed per epoch.
+
+```go
+    func GetLPRewardsPoolAddress(poolId uint64) {
```

Comment:
This function should be in Liquidity pool keeper. right?

Reply:
This would be good to put on incentive types folder - since this is related to rewards and this will need to be used on margin as well not just on amm module.

Reply:
How can he know unless access to liquidity pool module?